web/impact/newrelic.ini

```diff
@@ -209,4 +209,8 @@ monitor_mode = true
 app_name = MC Impact API Production
 monitor_mode = true
 
+[newrelic:temp]
+app_name = MC Impact API Temp (Production DB)
```

Comment:
This is utter bikeshedding (i.e. non-blocking), but I'd rather see it called just "MC Impact API Temp". I can't think of how that parenthetical will be useful, but I *can* imagine it being confusing if the `temp` host sticks around longer than this ticket's project.

Reply:
done